metsim/metsim.py

```diff
@@ -74,6 +66,12 @@ class MetSim(object):
         "base_elev": 0,
         "t_max_lr": '',
         "t_min_lr": '',
+        "sw_prec_thresh": 0.0,
+        "mtclim_swe_corr": False,
+        "lw_cloud": 'cloud_deardorff',
+        "lw_type": 'prata',
+        "tdew_tol": 1e-3,
+        "tmax_daylength_fraction": 0.67,
```

Comment:
These are not really "constants" but rather parameters that should be able to be modified by the configuration file.  Putting them here ties them to a specific run (a set of parameters defines the `MetSim` object).  These aren't really hardcoded, but rather defaults.  If you look in the `Metsim` constructor  you'll see that the given parameters are updated then.